fitbenchmarking/utils/options.py

```diff
@@ -176,7 +176,7 @@ class Options:
                 'OUTPUT': DEFAULT_OUTPUT,
                 'LOGGING': DEFAULT_LOGGING}
 
-    def __init__(self, file_name=None, results_directory: str = ""):
+    def __init__(self, additional_options = {}, file_name=None, results_directory: str = ""):
```

Comment:
The style checker is upset with a default argument of `{}`. The details of this are a bit murky, but it doesn't like any object as a default argument, it's much happier with a literal.

The standard way to get around this is have a default argument of `None`, and then overwrite this if required:
```
def some_function(arg1, arg2={}, arg3=None):
    """
    arg2 and arg3 are both initialied to an empty dict if no value is given.
    The style checker will complain about arg2. arg3 is safer.
    """
    if arg3 is None:
        arg3 = {}
    ...
```

I also think additional_options should be after file_name